spanner/Gemfile.lock

```diff
@@ -3,11 +3,11 @@ GEM
   specs:
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    concurrent-ruby (1.1.6)
```

Comment:
nit: should library upgrades be part of the same PR or could we push them out separately?

Reply:
I only upgraded `google-cloud-spanner` to the latest version, which is needed for the code samples. Usually, I do it together and I would not worry too much about it.